python/servo/devenv_commands.py

```diff
@@ -67,11 +70,61 @@ def cargo_update(self, params=None, package=None, all_packages=None):
         help='Updates all packages. NOTE! This is very likely to break your ' +
              'working copy, making it impossible to build servo. Only do ' +
              'this if you really know what you are doing.')
-    def update_cargo(self, params=None, package=None, all_packages=None):
+    @CommandArgument(
+        '--dry-run', '-d', action='store_true',
+        help='Show outdated packages.')
+    def update_cargo(self, params=None, package=None, all_packages=None, dry_run=None):
         if not params:
             params = []
 
-        if package:
+        if dry_run:
+            import toml
+            import json
+            import httplib
+            import colorama
+
+            content = toml.loads(open("./Cargo.lock").read())
```

Comment:
Shouldn't we be using `self.context.topdir` for this? This could break at some point of time, right? Also, we could utilize `toml.load` here, which parses directly from a descriptor.